pimd/pim_cmd.c

```diff
@@ -6134,8 +6121,6 @@ void pim_cmd_init()
   install_element (CONFIG_NODE, &debug_msdp_packets_cmd);
   install_element (CONFIG_NODE, &no_debug_msdp_packets_cmd);
   install_element (CONFIG_NODE, &undebug_msdp_packets_cmd);
-  install_element (CONFIG_NODE, &ip_msdp_peer_cmd);
-  install_element (CONFIG_NODE, &no_ip_msdp_peer_cmd);
```

Comment:
I see these as DEFUN_HIDDEN elsewhere in pim_cmd.c and don't see them installed anywhere else but here; remove entirely?